velox/benchmarks/tpch/TpchBenchmark.cpp

```diff
@@ -277,6 +290,9 @@ BENCHMARK(q22) {
 }
 
 int main(int argc, char** argv) {
+  std::string usage(
+      "This program benchmarks TPC-H queries. Run 'velox_tpch_benchmark -helpon=Tpchbenchmark' for available options.\n");
```

Comment:
@majetideepak Thank you for these clarifications.

nit: usage -> static ... kUsage

-helpon=Tpchbenchmark -> -helpon=TpchBenchmark